integration-tests/suites/missing_proc_scrape.go

```diff
@@ -12,15 +12,25 @@ type MissingProcScrapeTestSuite struct {
 	IntegrationTestSuiteBase
 }
 
+const fakeProcDir = "/tmp/fake-proc"
+
 func (s *MissingProcScrapeTestSuite) SetupSuite() {
-	_, err := os.Stat("/tmp/fake-proc")
-	assert.False(s.T(), os.IsNotExist(err), "Missing fake proc directory")
```

Comment:
I'm pretty sure this check was here because a script was needed to be run before the test so the `/tmp/fake-proc` directory was setup, is that not needed anymore?

Reply:
Actually, yeah, line 27 clearly states the directory should be created by the `create-fake-proc.sh` script.

Reply:
As discussed, this is a weird one, because the script is never called anywhere and os.Stat should never have worked because we have historically operated the tests from a different host to the one under test. Not sure how this ever worked.